packages/react/src/components/Alert/Alert.tsx

```diff
@@ -94,10 +107,28 @@ class Alert extends UIComponent<WithAsProp<AlertProps>, AlertState> {
   }
 
   renderContent = ({ styles, accessibility }: RenderResultConfig<AlertProps>) => {
-    const { action, content } = this.props
+    const { action, icon, header, content } = this.props
 
     return (
       <>
```

Comment:
alertWarningBehavior adds aria-live to the content slot of the Alert which makes the screen reader read the content. Since now we have header with the textual information as well, it would be good to read that as well. However, that would require us to have nested slots - let's discuss how to deal with it. fyi @layershifter